### PR TITLE
Adds special thanks section to README that recognizes test dependencies

### PR DESCRIPTION
**Pull request by morganzellers**
## Checklist:
- [x] Did you sign the [Contributor License Agreement](https://cla-assistant.io/wwt/SwiftCurrent)?
- [x] Is the linter reporting 0 errors?
- [x] Does the CI pipeline pass?
- [x] Did you [update the documentation](https://github.com/wwt/SwiftCurrent/blob/main/.github/CONTRIBUTING.md#documentation)?

### CODE REVIEW

**Comment by morganzellers** [comment]
Draft only until the verbiage-riffing (if any) has subsided

File: README.md [hunk 1]
```diff
@@ -127,6 +127,10 @@ And just like that you're started!
 - [Upgrade Path](https://github.com/wwt/SwiftCurrent/blob/main/wiki/UPGRADE_PATH.md)
 - [Contributing to SwiftCurrent](https://github.com/wwt/SwiftCurrent/blob/main/.github/CONTRIBUTING.md)
 
+# Special Thanks
```
Comment (Tyler-Keith-Thompson) [inline README.md:130]: My $0.02 is that we should move this below feedback. Also, why not just go ahead and call out the other dependencies?

Comment (Tyler-Keith-Thompson) [reply README.md:130]: Consider making it clear these are test-only dependencies. It's a bonus for us that people recognize we do not have any production dependencies outside of Apple 1st party frameworks.

Comment (morganzellers) [reply README.md:130]: I like the suggestion to include the others and specify as test-only. I had the section below feedback at first and I'm thinking I like that placement more as well.  - will update!

**Review by Tyler-Keith-Thompson — APPROVED** [review]

**Review by wiemerm — APPROVED** [review]

**Pull request merged**